storage/time-series/influxdb/storer/app/influx/Writer.js

```diff
@@ -0,0 +1,185 @@
+const { InfluxDB, Point } = require('@influxdata/influxdb-client');
+const { Logger } = require('@dojot/microservice-sdk');
+const { parseDateTimeToUnixNs } = require('../Utils');
+
+const logger = new Logger('influxdb-storer:influxdb/WriterData');
+/**
+ * This class handle with writer data in a specific bucket.
+ * @class
+ */
+class Writer {
+  /**
+   *
+   * @param {String} url   Url to access influxdb
+   * @param {String} token  A token with write permission in all orgs
+   * @param {String} defaultBucket  Bucket Name for all data write
+   * @param {Object} writeOptions  Default {} (See more at https://influxdata.github.io/influxdb-client-js/influxdb-client.writeoptions.html)
+   *
+   * @throws If the value for `writeOptions.flushInterval` must be greater than 0
+   */
+  constructor(url, token, defaultBucket, writeOptions = {}) {
+    logger.debug('constructor:');
+    logger.debug(`constructor: url=${url}`);
+    logger.debug(`constructor: token=${token}`);
+    logger.debug(`constructor: defaultBucket=${defaultBucket}`);
+    logger.debug('constructor: writeOptions=', writeOptions);
+    this.bucket = defaultBucket;
+    this.precision = 'ns';
+    this.writeOptions = writeOptions;
+
+    if (this.writeOptions.flushInterval && this.writeOptions.flushInterval <= 0) {
+      throw new Error('The value for `writeOptions.flushInterval` must be greater than 0');
+    }
+
+    this.writeOptions.writeFailed = (error, lines, attempts) => {
+      logger.error(`writeFailed: lines: ${lines.toString()} attempts:${attempts}`, error);
+    };
+
+    this.writeOptions.writeSuccess = (lines) => {
+      logger.debug(`writeSuccess: lines: ${lines.toString()}`);
+    };
+
+    this.influxDB = new InfluxDB({ url, token });
+
+    /**
+     * Map instances for WriteApi for each org
+     */
+    this.writers = new Map();
+  }
+
+  /**
+   * Get an instance of  WriteApi for the supplied organization and bucket.
+   *
+   * @param {string} org Organization Name
+   *
+   * @throws If Cannot get Writer for a org
+   */
+  getWriter(org) {
```

Comment:
not to be confused with the class name itself, could this method be called `getWriteAPI()` or `getAPI()`?